packages/react-native-aztec/src/AztecView.js

```diff
@@ -189,10 +189,13 @@ class AztecView extends Component {
 
 	blur() {
 		AztecInputState.blur( this.aztecViewRef.current );
+		// Unmounting the component needs to update the current focused element
+		AztecInputState.notifyInputChange();
```

Comment:
I thought this won't be necessary as the `AztecView` component would be notified when the blur event is triggered:

https://github.com/WordPress/gutenberg/blob/edd61d897241374b0558c8f884c8b31ef7a65008/packages/react-native-aztec/src/AztecView.js#L135-L147

If that's not the case, I'm wondering if we should call `notifyInputChange` directly within the `blur` and `focus` functions in `AztecInputState`, instead of here, wdyt?

https://github.com/WordPress/gutenberg/blob/edd61d897241374b0558c8f884c8b31ef7a65008/packages/react-native-aztec/src/AztecInputState.js#L98-L105

Reply:
The thing is in some cases `blur` gets called directly so `_onBlur` would never be called. I like the suggestion of moving it to be within the `blur` function in `AztecInputState`, I'll update it 👍

Reply:
Updated it in https://github.com/WordPress/gutenberg/pull/40788/commits/695a65bccba66afa9878d7ee14a360163323a932